AndroidSDKTests/src/test/java/com/leanplum/utils/BitmapUtilTest.java

```diff
@@ -96,11 +96,11 @@ public void setUp() {
   public void getScaledBitmapTest() throws Exception {
     Bitmap bitmap = Bitmap.createBitmap(1000, 400, Bitmap.Config.ARGB_8888);
     assertNotNull(bitmap);
-    when(BitmapUtil.class, "getBitmapFromUrl", "test", 480, 289).thenReturn(bitmap);
+    when(BitmapUtil.class, "getBitmapFromUrl", "test", 320, 193).thenReturn(bitmap);
     Bitmap scaledBitMap = BitmapUtil.getScaledBitmap(context, "test");
     assertNotNull(scaledBitMap);
-    assertEquals(289, scaledBitMap.getHeight());
-    assertEquals(480, scaledBitMap.getWidth());
+    assertEquals(193, scaledBitMap.getHeight());
+    assertEquals(320, scaledBitMap.getWidth());
```

Comment:
You should put these magic numbers into variables so you don't have to change them in multiple places.